src/main/java/mekanism/common/integration/computer/ComputerMethodFactory.java

```diff
@@ -80,6 +81,10 @@ void bindTo(@Nullable T subject, BoundMethodHolder holder) {
         }
     }
 
+    public List<MethodHelpData> getHelpData() {
+        return this.methods.stream().sorted(METHODDATA_COMPARATOR).map(MethodHelpData::from).toList();
```

Comment:
Given how this method is used in `FactoryRegistry#getHelpData` I believe this is intended to be using `collect(Collectors.toList())` rather than `toList()` as it seems like the desired outcome is for the returned list to be mutable.